Pipfile

```diff
@@ -4,10 +4,11 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-sceptre = "2.4.0"
+sceptre = "2.5.0"
```

Comment:
shouldn't these be in `dev-packages`?

Reply:
It could be in dev, but then the only contents of the Pipfile would be in dev.